packages/firestore/src/remote/serializer.ts

```diff
@@ -262,8 +262,14 @@ export function fromBytes(
     return ByteString.fromBase64String(value ? value : '');
   } else {
     hardAssert(
-      value === undefined || value instanceof Uint8Array,
-      'value must be undefined or Uint8Array'
+      value === undefined ||
+        // The Buffer interface extends the Uint8Array interface, however
```

Comment:
Optional: Re-write this comment as if it were a "command" to the JavaScript runtime. Inline comments written in this fashion tend to be maximally readable.

Here is one suggestion:

Check if the value is an instance of _both_ `Buffer` and `Uint8Array`, despite the fact that `Buffer` _extends_ `Uint8Array` because in some environments, such as jsdom, the prototype chain of `Buffer` unexpectedly does not indicate that it extends `Uint8Array`.

Reply:
Thank you. Nice suggestion for improvement.